lib/rubocop/cop/lint/empty_conditional_body.rb

```diff
@@ -107,13 +107,41 @@ def correct_other_branches(corrector, node)
           end
         end
 
+        def correct_other_branches?(node)
+          return false unless node.if_type? && node.else_branch
+
+          !(!empty_if_branch?(node) && node.elsif?) && !empty_else_branch?(node)
```

Comment:
Can you refactor to the below? Because this condition is complicated.

```suggestion
          return false unless node.if_type? && node.else_branch
          return false if !empty_if_branch?(node) && node.elsif?

          !empty_else_branch?(node)
```